<launch>

    <node name="tracking_target_node" pkg="puntilt_control" type="tracking_target.py" output="screen">
        <param name="pan_joint_name"  value="base_puntilt_joint"   />
        <param name="tilt_joint_name" value="base_realsense_joint" />
        <rosparam param="pan_joint_limit"> {min: -1.57, max: 1.57} </rosparam>
        <rosparam param="tilt_joint_limit"> {min: -1.57, max: 1.57} </rosparam>
        <param name="p_gain" value="0.075" />
        <param name="d_gain" value="0.05" /> <!-- no need in almost situations -->
        <param name="i_gain" value="0.01" /> <!-- no need in almost situations -->
        <param name="step_time" value="1.0" /> <!-- [sec] fast -> set small value -->
    </node>

</launch>
